project/Dependencies.scala

```diff
@@ -47,6 +47,7 @@ object Dependencies {
     val igluCore         = "0.2.0"
     // Scala (test only)
     val specs2           = "4.1.0" //"3.9.2"
+    val kafkaV           ="2.0.0"
```

Comment:
is this compatible with the Kafka we are using on Kubernetes? Just to make sure that we use the latest Kafka client version which goes with the Kafka we have installed.